nifi-api/src/main/java/org/apache/nifi/authorization/Group.java

```diff
@@ -24,7 +24,7 @@
 /**
  * A group that users can belong to.
  */
-public class Group {
+public class Group { // TODO rename to UserGroup
```

Comment:
I didn't want to cause too much of a ripple effect of changes outside the scope of the JIRA, but I can make those changes in this PR.